### ISSUE TITLE
Webpack issue - "Critical dependency: require function is used in a way in which dependencies cannot be statically extracted"

### ISSUE THREAD

**Issue by manniL** [andrusieczko/clip-path-polygon#28]
I'm using webpack to convert my ES6 JS to "universal" JS code.

```
window.$ = window.jQuery = require('jquery');
//...
require('clip-path-polygon');
```

When first requiring jquery and then requiring clip-path-polygon, webpack throws the following error:

`Critical dependency: require function is used in a way in which dependencies cannot be statically extracted`

It looks like the reason is line 11. When removing the require statements, the error vanishes.
```
var jQuery = jQuery || globalVariable.jQuery || (require && require('jquery'));
```

BTW: Changing the order of require statements does not change anything.

**Comment by andrusieczko** [comment]
Hi,

Is it possible that you give me an example of your webpack config / your example project?
It would be much easier for me to fix!

Thanks!
Karol

**Comment by manniL** [comment]
@andrusieczko Sure! I'm using Laravel (and therefore Laravel Mix that is built on top of webpack), so my config is [this one](https://github.com/JeffreyWay/laravel-mix/blob/master/setup/webpack.config.js).

**Comment by manniL** [comment]
@andrusieczko Any updates or ideas? Is there any reason to not remove the `|| (require && require('jquery'));` statement?

**Comment by andrusieczko** [comment]
Hi,
I simply didn't have time to build a simple project with webpack yet.
I've never used webpack, so it's not gonna be 5 minutes, sorry for that.
If you have some time to provide me with a basic project with clip-path-polygon dependency only, that would definitely help me solve the problem much faster.
Thanks!

**Comment by manniL** [comment]
No problem, I've created a [repository](https://github.com/manniL/webpack-polyong-test-project) with an example project so you can fix the error faster!

Steps to reproduce:

1. `npm install`
2. `webpack`

**Comment by andrusieczko** [comment]
Merged & published the version 0.1.12 to npm.

Thank you very much!

**Comment by manniL** [comment]
@andrusieczko Thanks a lot :) Happy I could help :smile:
